src/js/modules/authentication/actions.js

```diff
@@ -32,12 +32,17 @@ function requestUserData() {
   };
 }
 
-export function successLogin(response) {
+export function successLogin(token) {
+  try {
+    localStorage.setItem('jwt', token);
+  } catch (err) {
+    // do nothing
```

Comment:
We don't need to do anything if writing fails, but if we don't catch it then nothing after the setItem continues if an error occurs. An error can occur here if local storage is full, which I think safari incognito mode will throw.